packages/dds/map/src/map.ts

```diff
@@ -202,23 +202,22 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
 		//    and result in non-incremental snapshot.
 		//    This can be improved in the future, without being format breaking change, as loading sequence
 		//    loads all blobs at once and partitioning schema has no impact on that process.
-		for (const key of Object.keys(data)) {
-			const value = data[key];
-			if (value.value && value.value.length >= MinValueSizeSeparateSnapshotBlob) {
+		for (const [key, { value, type }] of Object.entries(data)) {
```

Comment:
Probably a good change, but out of curiosity what effect did turning noUncheckedIndexedAccess have on this?

Reply:
Accessing an index of an array or object will start to error and since we are accessing `data[key]`, that part of the code will be undefined